ui/intermittents-view/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { render } from "react-dom";
+import { AppContainer } from "react-hot-loader";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "font-awesome/css/font-awesome.css";
+import "../css/treeherder-global.css";
```

Comment:
Could you add a newline between the third party package imports and the treeherder ones with relative paths? (We've just started using this convention in the last few days, so don't expect you to have known hehe. There's a new eslint rule we could enable that I'm going to check out soon)